### PR TITLE
[zh-cn] sync consistent-list-from-cache.md

### PR DESCRIPTION
**Pull request by qlijin**
sync file:
```
content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md
```

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md [hunk 1]
```diff
@@ -9,8 +9,27 @@ stages:
...
+To consistent list from cache Kubernetes requires a newer etcd version (v3.4.31+ or v3.5.13+),
+that includes fixes to watch progress request feature.
+If older etcd version is provided Kubernetes will automatically detect it and fallback to serving consistent reads from etcd.
+Progress notifications ensure watch cache is consistent with etcd while reducing
+the need for resource-intensive quorum reads from etcd.
+
+See the Kubernetes documentation on [Semantics for **get** and **list**](/docs/reference/using-api/api-concepts/#semantics-for-get-and-list) for more details.
 -->
-允许 API 服务器从缓存中提供一致的 list 操作。
+通过直接从监视缓存中提供一致的 **list** 请求，提升 Kubernetes API 服务器的性能，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md:28]:
```suggestion
通过直接使用监视缓存来为 **list** 请求提供一致性的数据，提升 Kubernetes API 服务器的性能，
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md [hunk 2]
```diff
@@ -9,8 +9,27 @@ stages:
...
+从而改善可扩展性和响应时间。为了从缓存获取一致的列表，Kubernetes 需要使用较新的
+Etcd 版本（v3.4.31+ 或 v3.5.13+），这些版本包含了对监视进度请求功能的修复。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md:30]:
```suggestion
Etcd 版本（v3.4.31+ 或 v3.5.13+），这些版本包含了对监视进度请求特性的修复。
```

**Comment by windsonsea** [comment]
Hi @qlijin,  please [sqush your commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits) into a single one before merging.

**Comment by qlijin** [comment]
Thank you. It's done

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**